### ISSUE TITLE
Build Error - Requires a placeholder substitution but no value for <onesignal_google_project_number> and <onesignal_app_id>

### ISSUE THREAD

**Issue by danielgomezrico** [OneSignal/OneSignal-Flutter-SDK#324]
# Versions
```YAML
environment:
  sdk: 2.10.4
  flutter: 1.22.4

dependencies:
-   onesignal_flutter: 2.6.2 and 2.6.1
```

# Description
Error:

```bash
mypath/android/app/src/main/AndroidManifest.xml:49:9-51:51 Error:
        Attribute meta-data#onesignal_app_id@value at AndroidManifest.xml:49:9-51:51 requires a placeholder substitution but no value for <onesignal_app_id> is provided.
mypath/android/app/src/main/AndroidManifest.xml:53:9-55:70 Error:
        Attribute meta-data#onesignal_google_project_number@value at AndroidManifest.xml:53:9-55:70 requires a placeholder substitution but no value for <onesignal_google_project_number> is provided.

FAILURE: Build failed with an exception.
```

This happens after trying:

```bash
flutter build appbundle --release

// Or
flutter build apk --release
```

**Comment by jkasten2** [comment]
@danielgomezrico Thanks for reporting, this shouldn't be happening as [this plugin should automatically being filling these with blank values ](https://github.com/OneSignal/OneSignal-Flutter-SDK/blob/2.6.2/android/build.gradle#L64)to pass the validation as they will be defined at runtime instead.

In the upcoming 3.0.0 version these build time parameters are going away so this won't be an issue in the future.

Today you can work around this issue by following step 2.3 from the OneSignal-Android-SDK setup guide.
https://documentation.onesignal.com/docs/android-sdk-setup-300#step-2---add-onesignal-dependencies

**Comment by danielgomezrico** [comment]
@jkasten2 is there any update on this release?

**Comment by leossmith** [comment]
I am having the same issues any update on when to expect the new version?

**Comment by nan-li** [comment]
The major release for `onesignal_flutter` has been released since [version 3.0.0](https://github.com/OneSignal/OneSignal-Flutter-SDK/releases/tag/3.0.0). I will close this issue.
